### ISSUE TITLE
How to hide Armor in 1.17+? (Trinkets API)

### ISSUE THREAD

**Issue by millennIumAMbiguity** [apace100/cosmetic-armor#41]
In the version with Curios API, I used it primarily to hide armor. In 1.18 I can't find a way to hide my armor. Is this feature removed?

**Comment by SaNch0sE** [comment]
I have the same question! Tried searching in keybinds - no luck

**Comment by AyUpItsAli** [comment]
Also looking for a way to do this

**Comment by madilizxd** [comment]
I'm having this same issue. I downloaded the mod to be able to hide the armor but I can't figure out how to do it in 1.20.1 Fabric

**Comment by nahia95** [comment]
it looks like it's still not posible to hide the armor...
anyone knows a workaround?

**Comment by ElGatoSam** [comment]
yeah looks like we dont have an opption like in show me your skin or smth
